Apps/Src/Task_Idle.c

```diff
@@ -0,0 +1,8 @@
+/* Copyright (c) 2020 UT Longhorn Racing Solar */
+void Task_Idle(void *p_arg) {
+    (void)p_arg;
+
+    while(1) {
+        // Todo: Do something cool or be lame.
```

Comment:
Are we doing something cool, or should this TODO be removed?

Reply:
I don't have any plans for this. Do you have any @manthanand

Reply:
I don't think we have time for something cool anymore.